cmd/virtual-kubelet/commands/root/root.go

```diff
@@ -140,13 +141,17 @@ func runRootCommand(ctx context.Context, c Opts) error {
 		"watchedNamespace": c.KubeNamespace,
 	}))
 
+	var leases v1beta1.LeaseInterface
+	if c.EnableNodeLease {
+		leases = client.CoordinationV1beta1().Leases(corev1.NamespaceNodeLease)
```

Comment:
Suggestion: `leaseClient` as `leases` is unclear to me.

Reply:
Done.